### ISSUE TITLE
0.2.175 breaks compatibilty with Firefox 60.x ESR

### ISSUE THREAD

**Issue by sklages** [miso-lims/miso-lims#1923]
Just a remark: I have upgraded our MISO to current version `0.2.175`. 

The installed Firefox version on all our windows systems is always current ESR (Extendend Support Release), which is `60.6`.

With this version all the elements in Sample, Libs etc. don't show up. Please see attached image.

![miso_ff60](https://user-images.githubusercontent.com/1837422/57462996-37d82700-727a-11e9-8c44-3b803bd7ff27.jpg)

Is there something special that has been changed which prevents FF60.x to show the web elements correctly?

**Comment by djcooke** [comment]
I'm looking into this now. Could you try doing a hard refresh to see if that makes a difference? Ctrl+Shift+R should do a hard refresh in Firefox for Windows.

Is it only the Edit Sample page that is affected? If the hard refresh doesn't help, can you open the Firefox developer tools by pressing F12 and then reload the page, and let me know if any Javascript errors show up?

**Comment by sklages** [comment]
Hard reset didn't help. 

Affected are "Edit Sample", "Edit Library", "Edit Pool".

There is an error showing up:
```
TypeError
​
columnNumber: 18
​
fileName: "http://miso.molgen.mpg.de/scripts/header_script.js?version=0.2.175"
​
lineNumber: 2877
​
message: "sections.flatMap is not a function"
​
stack: "findField@http://miso.molgen.mpg.de/scripts/header_script.js?version=0.2.175:2877:18\nupdateField@http://miso.molgen.mpg.de/scripts/header_script.js?version=0.2.175:2681:21\nonChange@http://miso.molgen.mpg.de/scripts/header_script.js?version=0.2.175:4334:21\ncreateForm/</<@http://miso.molgen.mpg.de/scripts/header_script.js?version=0.2.175:2734:13\ncreateForm/<@http://miso.molgen.mpg.de/scripts/header_script.js?version=0.2.175:2732:9\ncreateForm@http://miso.molgen.mpg.de/scripts/header_script.js?version=0.2.175:2731:7\n@http://miso.molgen.mpg.de/miso/sample/13125:211:5\nl@http://miso.molgen.mpg.de/scripts/jquery/js/jquery-1.8.3.min.js:2:16991\nfireWith@http://miso.molgen.mpg.de/scripts/jquery/js/jquery-1.8.3.min.js:2:17783\nready@http://miso.molgen.mpg.de/scripts/jquery/js/jquery-1.8.3.min.js:2:12502\nA@http://miso.molgen.mpg.de/scripts/jquery/js/jquery-1.8.3.min.js:2:9907\n"
​
__proto__: Object { stack: "", … }
header_script.js:20113:3
TypeError: sections.flatMap is not a function[Learn More]
header_script.js:2877:18
findField
http://miso.molgen.mpg.de/scripts/header_script.js:2877:18
updateField
http://miso.molgen.mpg.de/scripts/header_script.js:2681:21
onChange
http://miso.molgen.mpg.de/scripts/header_script.js:4334:21
createForm/</<
http://miso.molgen.mpg.de/scripts/header_script.js:2734:13
forEach self-hosted:261:13 createForm/<
http://miso.molgen.mpg.de/scripts/header_script.js:2732:9
forEach self-hosted:261:13 createForm
http://miso.molgen.mpg.de/scripts/header_script.js:2731:7
<anonymous>
http://miso.molgen.mpg.de/miso/sample/13125:211:5
l
http://miso.molgen.mpg.de/scripts/jquery/js/jquery-1.8.3.min.js:2:16991
fireWith
http://miso.molgen.mpg.de/scripts/jquery/js/jquery-1.8.3.min.js:2:17783
ready
http://miso.molgen.mpg.de/scripts/jquery/js/jquery-1.8.3.min.js:2:12502
A
http://miso.molgen.mpg.de/scripts/jquery/js/jquery-1.8.3.min.js:2:9907
```
Does this help?

**Comment by djcooke** [comment]
Yes, that helped a lot, thanks! It seems flatmap isn't available until Firefox 62. PR https://github.com/miso-lims/miso-lims/pull/1926 fixes the issue, and should be included in v0.2.177. You'll want to wait until then to upgrade, as v0.2.176 will cause the same issue to appear on several additional forms.
